README.md

```diff
@@ -84,15 +48,15 @@ LogWeasel.configure do |config|
 end
 ```
 
-`config.key`  (default is the Rails `app_name`)  
-A string that will be included in your transaction IDs and is particularly useful in an environment where a unit of work may span multiple applications.  
+`config.key` (default is the Rails `app_name`)
+A string that will be included in your transaction IDs and is particularly useful in an environment where a unit of work may span multiple applications.
+
+`config.disable_delayed_job_tracing` (default is `false`)
+A boolean that disables Log Weasel appending a `log_weasel_id` parameter in the payloads of delayed Resque jobs. The default is `false`.
 
-`config.disable_delayed_job_tracing` (default is `false`)  
-A boolean that disables Log Weasel appending a `log_weasel_id` parameter in  the payloads of delayed Resque jobs. The default is `false`. 
+`config.debug` (default is `false`)
+A boolean that enables some additional debug logging. The default is `false`.
```

Comment:
i'm not sure what the real change was here...

Reply:
This is all whitespace changes (thanks to prettier).